lms/templates/main.html

```diff
@@ -116,16 +98,24 @@
   <meta name="path_prefix" content="${EDX_ROOT_URL}">
   <meta name="google-site-verification" content="_mipQ4AtZQDNmbtOkwehQDOgCxUUV2fb_C0b6wbiRHY" />
 
-  <%include file="${google_analytics_file}" />
-
-% if style_overrides_file:
-  <link rel="stylesheet" type="text/css" href="${static.url(style_overrides_file)}" />
+<% ga_acct = microsite.get_value("GOOGLE_ANALYTICS_ACCOUNT", settings.GOOGLE_ANALYTICS_ACCOUNT) %>
```

Comment:
It looks like it is needed by microsites... it seems a shame to have microsite-specific logic live in the main template. Isn't the goal that microsites can own their own logic. Is the problem that there isn't a 'microsite' theme that individual microsites override? I can see that this would mean we'd end up duplicating the logic into each microsite if that was the case.